make -fno-plt the default, results in much better branch prediction and code gen

--- gcc-12.1.0/gcc/common.opt~	2022-05-06 07:30:56.000000000 +0000
+++ gcc-12.1.0/gcc/common.opt	2022-12-20 15:20:48.817970699 +0000
@@ -2259,7 +2259,7 @@
 Generate position-independent code for executables if possible (small mode).
 
 fplt
-Common Var(flag_plt) Init(1) Optimization
+Common Var(flag_plt) Init(0) Optimization
 Use PLT for PIC calls (-fno-plt: load the address from GOT at call site).
 
 fplugin=
